firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionTesterApiClient.java

```diff
@@ -30,7 +30,7 @@
 
   private static final String TAG = "FADTesterApiClient";
   private static final String RELEASE_ENDPOINT_URL_FORMAT =
-      "https://firebaseapptesters.googleapis.com/v1alpha/devices/-/testerApps/%s/installations/%s/releases";
+      "https://firebaseapptesters.googleapis.com/v1alpha/devices123/-/testerApps/%s/installations/%s/releases";
```

Comment:
This should stay as devices becuase this is the name of the resource, not a specific ID